app/src/main/kotlin/com/wire/android/ui/home/newconversation/NewConversationViewModel.kt

```diff
@@ -195,7 +197,7 @@ class NewConversationViewModel @Inject constructor(
                     readReceiptsEnabled = groupOptionsState.isReadReceiptEnabled,
                     accessRole = groupOptionsState.accessRoleState,
                     access = if (groupOptionsState.accessRoleState.contains(Conversation.AccessRole.GUEST)) {
-                        setOf(Conversation.Access.INVITE, Conversation.Access.CODE)
+                        Conversation.defaultGroupAccess.toMutableSet().apply { add(Conversation.Access.CODE) }
```

Comment:
Doesn't `defaultGroupAccess` already include the `Access.CODE` flag?

Reply:
No, only INVITE